modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -146,24 +146,39 @@
         }
         'mobile-dep': {
             $google_play_config = {
+                'fenix'              => {
+                    service_account             => 'dummy',
+                    certificate                 => 'dummy',
+                    certificate_target_location => "${root}/fenix.p12",
+                },
                 'reference-browser'  => {
                     service_account             => 'dummy',
                     certificate                 => 'dummy',
-                    certificate_target_location => "${root}/dep.p12",
+                    certificate_target_location => "${root}/reference_browser.p12",
```

Comment:
We can't have two different products both with `certificate_target_location` with the same file (I guess obviously :stuck_out_tongue: whoops)

Reply:
Good catch!